blog/views.py

```diff
@@ -400,3 +400,10 @@ def change_password(request):
 
     context = {"form": form}
     return render(request, "blog/change_password.html", context)
+
+
+def notifications(request):
+    notifications = Notification.objects.filter(receiver_user=request.user)
+
+    context = {"notifications": notifications}
+    return render(request, "blog/notifications.html", context)
```

Comment:
The notification section only displays notifications addressed to the post author.